test/test/helpers/use-fake-dom/index.js

```diff
@@ -1,10 +1,10 @@
 import debug from 'debug';
 
-import reactTestEnvSetup from 'lib/react-test-env-setup';
+import reactTestEnvSetup from 'react-test-env';
 
 const log = debug( 'calypso:test:helpers:setup-dom-env' );
 
-export default function domWrapper( markup, features ) {
+export default function useFakedDom( markup, features ) {
```

Comment:
- From the folder name, I'd expect this to be `useFakeDom`
- Couldn't we still call on the original `reactTestEnvSetup.useFakeDom`, but include an additional `after` block for the `require.cache` cleanup?

```js
export default const useFakeDom = flow( [ reactTestEnvSetup.useFakeDom, () => {
 	after( () => delete require.cache[ require.resolve( 'page' ) ] );
} ] );
```